libraries/AP_Frsky_Telem/AP_Frsky_SPort_Passthrough.cpp

```diff
@@ -188,6 +190,16 @@ bool AP_Frsky_SPort_Passthrough::is_packet_ready(uint8_t idx, bool queue_empty)
     case BATT_2:
         packet_ready = AP::battery().num_instances() > 1;
         break;
+    case RPM:
+        {
+            packet_ready = false;
+            const AP_RPM *rpm = AP::rpm();
+            if (rpm == nullptr) {
+                break;
+            }
+            packet_ready = rpm->num_sensors() > 0;
```

Comment:
```suggestion
            packet_ready = AP::rpm() && APM::rpm()->num_sensors() > 0;
```

Reply:
@peterbarker I'm sure you told me once to not rely on implicit cast and and favor check vs nullptr instead, which btw I found pretty consistent as a suggestion, I did see in the code using the somewhat simpler implicit cast quite ofter though

Reply:
That's true.  Sometimes I just get lazy.

Could also have been
```
            packet_ready = (AP::rpm() != nullptr) && APM::rpm()->num_sensors() > 0;
```

Not quite as pithy :-)